internal/dag/builder.go

```diff
@@ -518,6 +521,18 @@ func (b *Builder) computeHTTPProxy(proxy *projcontour.HTTPProxy) {
 			svhost.Secret = sec
 			svhost.MinProtoVersion = annotation.MinProtoVersion(proxy.Spec.VirtualHost.TLS.MinimumProtocolVersion)
 
+			if proxy.Spec.VirtualHost.TLS.EnableFallbackCertificate && !isBlank(b.FallbackCertificate) {
```

Comment:
This would emit a warning for each dag rebuild. We actually don't have any logging in the builder, so we'd need to introduce that which is fine, but we'll spam the logs with this warning.

Reply:
If a proxy sets `EnableFallbackCertificate` but the fallback cert isn't configured, that should be an invalid proxy, and we should use the status update to communicate that.